### ISSUE TITLE
[Adaptive Cards: Select Host App]: Keyboard focus is landing twice on the 'Select Host App' combo box.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8178]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 22H2(OS Build 25247.1000)
Browser Version: Version 110.0.1543.0 (Official build) dev (64-bit)

**Repro Steps:**
1. Open the above URL in the browser.
2.  Press tab key and navigate to the 'select host app' combo box and observe the issue.

**Actual Result:**
While navigating to the 'Select host App' combo box keyboard focus is landing twice on the control. On first time focus is landing on the entire combo box including label and on pressing tab key second time it is landing on combo box control.

**Expected Result:**
Keyboard focus should not land twice on the same control. On pressing tab key first time focus should be land on the combo box only.

**Note:**
Same issue is repro through out the application for the combo box on header.

**User Impact:**
It will impact the keyboard user if the focus will land twice on the same controls as it will take extra effort for the user to navigate  on the controls.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B80A4F009-8781-4423-B3CD-2448EAF5BF62%7D&file=MAS%202.4.3%20%E2%80%93%20Focus%20Order.docx&action=default&mobileredirect=true

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

https://user-images.githubusercontent.com/95913748/208830581-012e4383-d8b6-43d3-8129-78a7312da44a.mp4

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is not repro, hence closing this issue.
